python/ray/data/dataset.py

```diff
@@ -451,6 +452,62 @@ def get_dataset_id(self) -> str:
         """
         return self._plan.get_dataset_id()
 
+    @PublicAPI(api_group=BT_API_GROUP)
+    def load_from_uris(
+        self,
+        uri_column: str,
+        *,
+        decode_fn: Optional[Callable[[io.RawIOBase], Iterator[DataBatch]]] = None,
+    ):
+        """Load binary data from a column of URIs.
+
+        Args:
+            uri_column: The name of the column containing the URIs.
```

Comment:
Please be sure to document what is the returned column type, and what is the name of the new column?